src/models/CardScheme.ts

```diff
@@ -0,0 +1,20 @@
+export interface CardScheme{
```

Comment:
Ik zou deze `interface` niet exporten, maar ipv daarvan een union van alle implementaties exporten. Dat zorgt ervoor dat je gebruik kan maken van een tagged unions in andere delen van de code. Lees voor meer info [dit stukje](https://www.typescriptlang.org/docs/handbook/advanced-types.html#discriminated-unions).